src/mackerel-container-agent-definition.ts

```diff
@@ -1,4 +1,3 @@
-import { AmazonLinuxGeneration } from "@aws-cdk/aws-ec2"
```

Comment:
Hmm, I forgot to remove this import in #63. (I wonder tslint didn't complain at that time 🤔 )